platform/centec/rules.mk

```diff
@@ -8,12 +8,10 @@ include $(PLATFORM_PATH)/libsaithrift-dev.mk
 include $(PLATFORM_PATH)/python-saithrift.mk
 include $(PLATFORM_PATH)/docker-ptf-centec.mk
 
-SONIC_ALL += $(SONIC_ONE_IMAGE) \
-             $(DOCKER_PTF_CENTEC) \
-             $(DOCKER_SYNCD_CENTEC_RPC)
```

Comment:
why comment out DOCKER_PTF_CENTEC and DOCKER_SYNCD_CENTEC_RPC?

Reply:
DOCKER_PTF_CENTEC and DOCKER_SYNCD_CENTEC_RPC depend on LIBSAITHRIFT_DEV_CENTEC